config/styles/blade-effects.md

```diff
@@ -16,6 +16,8 @@ Note that some of these EFFECTS may be included in features used by some prop fi
 The majority of these are self explanitory. 
 Just read EFFECT as "do something during_XXXX"
 
+*Historical tidbit*: Some effects where deprecated for consistency in naming or the introduction of new methods. As such, this documentation is relevant for OS 5.0 and up.
```

Comment:
I don't think this is really relevant.
Nobody cares about 4.x and older anyways.